README.md

```diff
@@ -1 +1,25 @@
-# sourcery-cypress
\ No newline at end of file
+# sourcery-cypress
+## _Tutorial_
+## How to?
+### 1. Clone via Sourcetree
+
+1. Click [Clone/New] button
+2. Copy paste "https://github.com/nikolaj-tolkaciov/sourcery-cypress.git" into [Source Path / URL] input
+3. Input your desired path to store the repository on your local machine into [Destination Path] input 
+4. Click the [Clone] button
+
+### 2. Install
+
+1. (Prerequisites) Install <a href="http://www.oracle.com/technetwork/java/javase/downloads/index.html">JDK</a> and <a href="https://nodejs.org/en/download/">NodeJS</a> 
+2. Navigate to your local repository directory that you selected in _part 1_, click on the address/navigation bar, type 'cmd' and click Enter (or just open your command line/terminal by clicking [Windows] keyboard key, typing "cmd" and then navigate to your local repository directory through the terminal: write ***cd C:/Users/....*** (insert your local repository directory))
```

Comment:
C:/Users/.... this might be not in users and not in C drive

Reply:
But of course